package.json

```diff
@@ -20,7 +20,7 @@
     "@vuepress/plugin-google-analytics": "^1.2.0",
     "@vuepress/shared-utils": "^1.2.0",
     "clipboard": "^2.0.4",
-    "replace": "^1.2.0",
+    "replace": "^1.1.5",
```

Comment:
@nannanli I don't know why these changes appeared in this file in the first place. I haven't worked on it.